api/swagger.yml

```diff
@@ -2241,6 +2250,25 @@ paths:
         default:
           $ref: "#/components/responses/ServerError"
 
+  /otf/diffs:
+    get:
+      tags:
+        - experimental
```

Comment:
Please check what having 2 tags does to method naming in our various languages.  AFAIR some of the code generators move APIs to different objects depending on their tags, others reflect tags in method names.

For instance AFAICS the Java codegen adds it to _both_ "experimental" **and** "otfDiff".  That's disappointing: it makes more sense to call it as `otfDiffApi.otfDiffs()` than as `experimental.otfDiffs()`, and then we lose the experimental tag.

Perhaps we don't want a tag here.  Even just document it as experimental in the description, and perhaps add `/experimental` to the start of the path.

@nopcoder any preferences?